docs/source/app-dev/bindings-java/quickstart/template-root/daml/Main.daml

```diff
@@ -8,7 +8,14 @@ import Daml.Script
 import Iou
 import IouTrade()
 
-initialize : Script [Party]
+data Parties = Parties
+  { alice: Party
+  , bob: Party
+  , usBank: Party
+  , eurBank: Party
+  }
+
+initialize : Script Parties
```

Comment:
my intention with this is to replace this line of the quickstart document

> You will see that the script you used to initialize the sandbox is an initial segment of the ``trade_test`` script.

with something like

> You will see that the script you used to initialize the sandbox is invoked as the first step of the  ``trade_test`` script.

This way we don't have to keep the initial segments identical (they have actually drifted apart quite a bit)